src/com/google/enterprise/adaptor/GsaFeedFileSender.java

```diff
@@ -107,9 +109,19 @@ private void buildPostParameter(StringBuilder sb, String name,
   private byte[] buildGroupsXmlMessage(String groupsource, String feedtype,
       String xmlDocument) {
     StringBuilder sb = new StringBuilder();
-    buildPostParameter(sb, "groupsource", "text/plain", groupsource);
-    buildPostParameter(sb, "feedtype", "text/plain", feedtype);
-    buildPostParameter(sb, "data", "text/xml", xmlDocument);
+    String ft = feedtype.toLowerCase(US);
+    if (ft.equals("full") || ft.equals("incremental")) {
```

Comment:
I think that PJ doesn't want to see unused values here ("full" and "replace"), so we'll just address this in gsafeed and reconcile them later if necessary.